src/snapshot.js

```diff
@@ -118,15 +121,20 @@ export function restorePlayerSnapshot(player, snapshotObject) {
           currentTime = player.currentTime();
         }
         player.currentTime(currentTime);
+        player.play();
```

Comment:
this seems like a significant code change 🤔

Reply:
This PR was from awhile ago and I don't remember why this is here so I'm removing it. Just don't want to lose the comments.